jwql/website/apps/jwql/templates/api_landing.html

```diff
@@ -0,0 +1,152 @@
+{% extends "base.html" %}
+
+{% block preamble %}
+
+	<title>API - JWQL</title>
+
+{% endblock %}
+
+{% block content %}
+
+    <main role="main" class="container">
+        <h2>Explore the JWQL API</h2><hr>
+        <p align="justify">
+            <ul>
+                <li><a href="#introduction">What is the JWQL API?</a></li>
+                <li><a href="#examples">Examples</a></li>
+                <li><a href="#api_list">List of Avaliable Services</a></li>
+                <li><a href="https://jwql.readthedocs.io/en/latest/index.html?highlight=api">Documentation</a></li>
```

Comment:
I think this is probably a better URL for this link: https://jwql.readthedocs.io/en/latest/website.html#api-views-py

Right now the docs are broken for those, but that is a separate issue that we can solve in a different PR.

Reply:
Got it, good idea.